package/motioneye/extractl.py

```diff
@@ -449,7 +449,7 @@ def dateInterval():
         'section': 'expertSettings',
         'reboot': True,
         'required': True,
-        'depends': ['dateMethod==http'],
+        'depends': ['dateMethod==(http'],
```

Comment:
This should read `'depends': ['dateMethod==(http|sntp)'],`

Reply:
Oups, I have no idea how this has slipped here. In any case, the date interval property is only used with HTTP method. SNTP is only used at startup, while NTP is responsible for maintaining the system date thereafter.